server/config/config.go

```diff
@@ -506,8 +506,10 @@ type ScheduleConfig struct {
 	// If the number of times a region hits the hot cache is greater than this
 	// threshold, it is considered a hot region.
 	HotRegionCacheHitsThreshold uint64 `toml:"hot-region-cache-hits-threshold,omitempty" json:"hot-region-cache-hits-threshold"`
-	// StoreBalanceRate is the maximum of balance rate for each store.
+	// StoreBalanceRate is deprecated, use DefaultStoreLimit instead.
 	StoreBalanceRate float64 `toml:"store-balance-rate,omitempty" json:"store-balance-rate"`
+	// DefaultStoreLimit limits the number of operators to be executed per minute for each store.
+	DefaultStoreLimit float64 `toml:"default-store-limit,omitempty" json:"default-store-limit"`
```

Comment:
Some users are already familiar with the previous naming, and such changes seem to be incompatible.

Reply:
What do you mean about the incompatiblity?

The old name can also be used as a deprecated option and the changes here are compatible.

If you mean you don't agree this PR, I have some different opinions.

First, let's get consensus about the premise.
* `store-balance-rate` is not a good enough name
* `default-store-limit` is a better name

Then, if you agree with above two conclusions, let me explain why I recommend to do this change.
* I don't think we should give up improvement because of old historical reasons.
* The users in the past are important, the users in the future are important too.
* Sometime, we should guide the users to improve their old habits.

This do not mean we don't care about the user's old habits, the old name is preserved here, and I just mark the old name as deprecated. I plan to remove the old name completely on next major version like 4.0 or 4.1, before that, users can use the old name as their wish.

Reply:
What if a user tuning the `store-balance-rate` to 60, and update from old version to the new version, will the new arguments `default-store-limit` be 60?

Reply:
Yes, it is.

Reply:
I have test:
old pd-server set to `store-balance-ratio` to  6234
restart with new pd-server, the out show:
```
./bin/pd-ctl config show |grep store
    "default-store-limit": 15,
    "max-store-down-time": "30m0s",
    "store-balance-rate": 6234,
```
Please add a test, and I don’t think there are too many problems with the original name. cc@ @disksing

Reply:
@shafreeck  PTAL @rleungx

Reply:
@nolouch That's weird, I think I have tested. I will add a unit test.

@disksing @rleungx What do you think about the name?

Reply:
No strong opinion, but I'd like to propose `schedule-rate-limit`...

Reply:
@nolouch How did your test? If you define `default-store-limit` and `store-balance-rate` at the same time, `default-store-limit` will be used and `store-balance-rate` is abandoned because the user is aware of the new config option. In the upgrading case, there is no `default-store-limit` defined, so `store-balance-rate` will be used.

Anyway, I will add a test.

Reply:
both of them are fine to me

Reply:
We can not infer the relationship between the `store limit` and `store-balance-rate` from its name. Why do you guys think `store-balance-rate` is good name?

Reply:
From `store-limit`, we cannot know what does it limit. While `balance-rate` is not accurate either, `balance` usually refers to `balance-leader` and `balance-region`, but the limit applies to other schedulers and checkers at the same time.

Reply:
@nolouch I added some test cases, and it works as expected.